.github/workflows/test.yml

```diff
@@ -38,6 +38,6 @@ jobs:
     - name: Add wasm32 target
       run: rustup target add wasm32-unknown-unknown
     - name: Build
-      run: cargo build --verbose
+      run: cargo build --verbose --features unstable
```

Comment:
not sure why this is `build`, can just be `cargo check`, also prob worth checking without unstable feature in this workflow, should be low additional cost